samples/CommunityToolkit.Maui.Sample/ViewModels/Views/Popup/ComplexPopupViewModel.cs

```diff
@@ -0,0 +1,21 @@
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+
+namespace CommunityToolkit.Maui.Sample.ViewModels.Views;
+
+public partial class ComplexPopupViewModel(IPopupService popupService) : ObservableObject
+{
+	readonly IPopupService popupService = popupService;
+	readonly INavigation navigation = Application.Current?.Windows[0].Page?.Navigation ?? throw new InvalidOperationException("Unable to locate INavigation");
```

Comment:
For multi window app, `Windows[0]` doesn't mean the current/focused Window. With that the popup can show in the wrong window.

Reply:
Noted! What's the best way to find the active window nowadays?

The good news is that this doesn't impact the sample app since this sample doesn't use Multi-Windows. We're good pushing this PR though, but I'm just curious how we find the active window?

Reply:
(I'm going to "resolve" this conversation to unblock + merge this PR)

Reply:
https://vladislavantonyuk.github.io/articles/How-to-Get-the-Active-Window-in-.NET-MAUI/